### PR TITLE
Changes to get LATEST_RELEASE version, Bump to v2.24

### PR DESCRIPTION
**Pull request by rkavalap**

### CODE REVIEW

**Comment by giggio** [comment]
We can't set it always to the latest release, as people installing 2.23 want that version, not latest. Otherwise the package version would be meaningless. We could do a dist-tag to do that, maybe. In that case, we need to structure the code in that way.

**Comment by rkavalap** [comment]
@giggio  one can set the npm_config_chromedriver_version variable if you look at my PR. That will override the LATEST scenario to the version a user wants to download.

**Comment by rkavalap** [comment]
I just rebased and pushed the latest changes.

**Comment by rkavalap** [comment]
I changed the default to helper.version. If someone want's the latest chromedriver_version from this package they can always set the CHROMEDRIVER_VERSION env variable to 'LATEST'

**Comment by giggio** [comment]
We need to support `https`, see where `requestOptions.protocol === 'https:' ? https : http` is used.
Also, we need to support proxies, see `getRequestOptions`.

**Comment by giggio** [comment]
@rkavalap I noticed you pushed a commit, are you done, can I verify?

**Comment by rkavalap** [comment]
@giggio Yes please.

File: install.js [hunk 1]
```diff
@@ -108,12 +117,12 @@ function findSuitableTempDirectory(npmConf) {
...
 
-function getRequestOptions(proxyUrl) {
+function getRequestOptions(proxyUrl, downloadPath) {
   var options
   if (proxyUrl) {
-    options = url.parse(proxyUrl)
-    options.path = downloadUrl
-    options.headers = { Host: url.parse(downloadUrl).host }
+    options = url.parse(proxyUrl) 
+    options.path = dowloadPath
```
Comment (giggio) [inline install.js]: There is a typo here on `dowloadPath`, should be `downloadPath`. It seems we are not testing install with a proxy... Anyway, please fix it.

**Comment by giggio** [comment]
@rkavalap It seems everything is working fine, there is only a problem, I made a comment in the code. Also, please update the readme file to include how to use the new functionality.

**Comment by rkavalap** [comment]
@giggio Fixed comments and an issue in the previous refactor and updated Readme.

**Comment by davscro** [comment]
Those changes are not merged, right?

**Comment by rkavalap** [comment]
@davscro yes, its not merged yet. @giggio is there something stopping for this to get merged ?

**Comment by giggio** [comment]
@rkavalap Sorry for the delay. I have integrated your changes on commit 2bab1d6822038674b51b896f857288798f8b8126. As there were conflicts, I already solved them. And I squashed them.
As the changes are already there, I am closing this PR.
It has been released as version `2.27.2`. It is up on npm already.

**Comment by rkavalap** [comment]
👍

**Pull request closed without merging**